### PR TITLE
[Feature] Onboarding screen accessibility traits

### PR DESCRIPTION
**Pull request by i336616**
## Checklist

* [x] Test your changes as thoroughly as possible before you commit them. Preferably, automate your test by unit/integration tests.
* [x] Set a speaking title. Format: {task_name} (closes #{issue_number}). For example: Use logger (closes # 41)

### CODE REVIEW

File: src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings [hunk 1]
```diff
@@ -458,6 +458,11 @@
 
 "ExposureNotificationSetting_Activate_OSENSetting_Description" = "For risk calculation, COVID-19 exposure notifications must be activated in the system settings so that exposures can be logged. Please activate COVID-19 exposure notifications in your system settings.";
 
+/* Home Navigation */
```
Comment (steins-code) [inline src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings:461]: Please dont modify english language files any more:)

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings [hunk 2]
```diff
@@ -467,6 +467,11 @@
 "ExposureNotificationSetting_AccLabel_BluetoothOff" = "Eine Person hat Bluetooth auf ihrem Smartphone ausgeschaltet, eine Begegnung mit zwei weiteren Personen wird daher nicht aufgezeichnet.";
 "ExposureNotificationSetting_AccLabel_InternetOff" = "Eine Person hat die Internetverbindung ihres Smartphones ausgeschaltet, eine Begegnung mit zwei weiteren Personen wird daher nicht aufgezeichnet.";
 
+/* Home Navigation */
```
Comment (steins-code) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:470]: isnt this change not already introduced by #282

File: src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings (hunk 1 above)
Comment (i336616) [reply src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings:461]: Apologies, I thought I'd reverted it already. I'll revert it back to the dev branch version now.

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings (hunk 2 above)
Comment (i336616) [reply src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:470]: It was, yes, I was working off these branches in parallel

**Review by steins-code — APPROVED** [review]

**Review by llanghammer-sap — APPROVED** [review]

**Pull request merged**